### PR TITLE
CI uses its own key-vault

### PR DESCRIPTION
**Pull request by mingwli**
[AB#379834](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/379834)

### CODE REVIEW

File: azure-pipelines-v3.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ pr:
 
 variables:
 - group: devresourcekeyvault
+- group: DocfxCIKVPub
 - name: dotnetVersion
```
Comment (herohua) [inline azure-pipelines-v3.yml:13]: We should use another KV in a specific tenant. Please check my comment under the feature.

**Review by herohua — CHANGES_REQUESTED** [review]
We should use another KV in a specific tenant. Please check my comment under the feature.

File: azure-pipelines-v3.yml (hunk 1 above)
Comment (mingwli) [reply azure-pipelines-v3.yml]: @herohua KV moved to PME

Comment (mingwli) [reply azure-pipelines-v3.yml]: @herohua KV moved to PME

**Review by herohua — APPROVED** [review]

**Pull request merged**